linode/manager
# fix: [M3-6606] - Add spacing between copy and Linode Rebuild button.

## Description 📝
Add's spacing between copy and Linode Rebuild button.

| Before  | After   |
| ------- | ------- |
|![image](https://github.com/linode/manager/assets/119517080/bcb9a9fd-ccec-4782-a531-d189bca4140f)| ![image](https://github.com/linode/manager/assets/119517080/2de9d92d-db28-4855-80e6-1531b6e65cfb) |

## How to test 🧪
### Verification steps
(How to verify changes)
- Navigate to Linode Rebuild form scroll down and observe the spacing for Rebuild Linode button.
- ...

## As an Author I have considered 🤔

*Check all that apply*
